File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.java
class Solution {
    public double average(int[] salary) {
        double ans = 0;
        Arrays.sort(salary);
        for(int i =1;i<salary.length-1;i++){
            ans+=salary[i];
            System.out.println(ans);
        }
        return ans/(salary.length-2);
    }
}

File: 1491-average-salary-excluding-the-minimum-and-maximum-salary/1491-average-salary-excluding-the-minimum-and-maximum-salary.js
/**
 * @param {number[]} salary
 * @return {number}
 */
var average = function(salary) {
    salary.sort(function(a,b){return a-b});
    console.log(salary);
    salary.shift();
    console.log(salary);
    salary.pop(salary.length-1);
    console.log(salary);
    return ans = salary.reduce((a,b)=>{return a+b})/salary.length;
};